tests/test_ingest.py

```diff
@@ -124,139 +126,146 @@ def test_ingest_dense_matrix(self):
         """Ingest Pipeline should extract, transform, and load dense matrices
         """
         args = [
-            '--study-id',
-            '5d276a50421aa9117c982845',
-            '--study-file-id',
-            '5dd5ae25421aa910a723a337',
-            'ingest_expression',
-            '--taxon-name',
-            'Homo sapiens',
-            '--taxon-common-name',
-            'human',
-            '--ncbi-taxid',
-            '9606',
-            '--genome-assembly-accession',
-            'GCA_000001405.15',
-            '--genome-annotation',
-            'Ensembl 94',
-            '--matrix-file',
-            'gs://fake-bucket/tests/data/dense_matrix_19_genes_1000_cells.txt',
-            '--matrix-file-type',
-            'dense',
+            "--study-id",
+            "5d276a50421aa9117c982845",
+            "--study-file-id",
+            "5dd5ae25421aa910a723a337",
+            "ingest_expression",
+            "--taxon-name",
+            "Homo sapiens",
+            "--taxon-common-name",
+            "human",
+            "--ncbi-taxid",
+            "9606",
+            "--genome-assembly-accession",
+            "GCA_000001405.15",
+            "--genome-annotation",
+            "Ensembl 94",
+            "--matrix-file",
+            "gs://fake-bucket/tests/data/dense_matrix_19_genes_1000_cells.txt",
+            "--matrix-file-type",
+            "dense",
         ]
         ingest = self.execute_ingest(args)[0]
         models = ingest.expression_ingestor.gene_docs
+        actual_data_arrays = ingest.expression_ingestor.data_array_docs
         # print(models)
         for model in models:
             # Ensure that 'ObjectID' in model is removed
-            del model['_id']
+            del model["_id"]
             # Verify gene model looks as expected
-            self.assertEqual(model, gene_models[model['name']])
+            self.assertEqual(model, gene_models[model["name"]])
+        for dr in actual_data_arrays:
+            del dr["linear_data_id"]
+            self.assertEqual(dr, data_arrays[dr["name"]])
```

Comment:
Out of curiosity, what does `dr` stand for?  Data row?

Reply:
Hey! This was actually was a typo on my end. I updated this values when I encountered a merge conflict. You can see the changes [here](https://github.com/broadinstitute/scp-ingest-pipeline/blob/4bc936b808a2778db8a3d7fb0b441df3d60fd963/tests/test_ingest.py#L163).